### DISCUSSION TITLE
Add a way to change config manually on files 🤔

### DISCUSSION THREAD

**Discussion by Hellsanx** [ppy/osu#28674] [Ideas] [answered]
This idea came out when I can't play after change renderer from Direct3D 11 to OpenGL (probably for science). My last decision was to delete everything include my 2 days process(cause I don't know where the config was save) and reinstall the game.

The idea is that if I can somehow change the renderer out of the game, I can avoid losing my process.

For example, I remember Minecraft have a config file that I can change it value outside the game setting menu.

**Comment by BlockHead007** [comment, marked as answer]
All configuration files can be found in data folder. Default data location is at %appdata%\osu. The file you needed is framework.ini, line Renderer =

**Reply by Hellsanx** [reply]
How the **** I don't see that when I'm digging the game folder 😲, btw thank you.
